src/bci_build/package/__init__.py

```diff
@@ -868,6 +874,10 @@ def dockerfile_from_line(self) -> str:
     def kiwi_derived_from_entry(self) -> str:
         if self._from_image is None:
             return ""
+        # Adjust for the absolutely braindamaged format that OBS expects to reference
```

Comment:
nit: let's not be that harsh.

```suggestion
        # Adjust for the weird format that OBS expects to reference
```